### PR TITLE
[upgrade] print update message before running the update code

### PR DESCRIPTION
**Pull request by MorrisJobke**
- otherwise you don't know which app update failed

cc @nickvergessen @LukasReschke @DeepDiver1975 @PVince81 

@DeepDiver1975 @cmonteroluque Decide on 8.1 or 8.2. I would then propose a backport to 8.1 and maybe more. cc @karlitschek

### CODE REVIEW

File: core/command/upgrade.php [hunk 1]
```diff
@@ -158,6 +158,9 @@ function () use($output, $updateStepEnabled, $self) {
 			$updater->listen('\OC\Updater', 'appUpgradeCheck', function () use ($output) {
 				$output->writeln('<info>Checked database schema update for apps</info>');
 			});
+			$updater->listen('\OC\Updater', 'appUpgradeStarted', function ($app, $version) use ($output) {
+				$output->writeln("<info>Update <$app> ...</info>");
```
Comment (nickvergessen) [inline core/command/upgrade.php]: I'd say `Updating <$app> ...`

**Comment by karlitschek** [comment]
I like this A LOT. Please put it in master and 8.1
We need to be way more verbose during upgrade.

**Comment by karlitschek** [comment]
:+1:

File: core/command/upgrade.php (hunk 1 above)
Comment (MorrisJobke) [inline core/command/upgrade.php]: Done

**Comment by butonic** [comment]
:+1:

**Pull request merged**